### ISSUE TITLE
How to become a freelancer, and how u got your first client ?

### ISSUE THREAD

**Issue by Daliprince** [anselmh/AMA#16]

**Comment by anselmh** [comment]
That’s a good question and not the shortest. But I’ll try to give you some insight.

I did my first websites as a pupil in secondary school (around 2003). Nothing special but around 2006 I got some requests by friends to make them a website. I didn’t want to give that away for free and also badly needed some money for my hobbies back then. 
Being aware that I can’t just receive money without having some sort of business I played around thinking how to solve the problems. A big problem was to get my parents on board because I was still not full age so needed the permission of them. 

Finally, I got it sorted out with a friend of mine at the age of 17 and started my first business in 2007. It was 25EUR and some weird bureaucratic things to manage but worth it. Our first client indeed was a relative of my friend who had a holiday residence so we made a website for him (for waaay to less money of course but we were newbies here).
There followed a few other projects for sport clubs (we always knew some people there). Being a bit bored of HTML (it was the times of frames and table layouts made with Photoshop), I dived into print design and did a couple of brochures, flyers and such stuff for the same and various other small clients. It was amazing to have my first real printed product in my hands which wasn’t a basic flyer (but a 64 pages brochure with all the tricky things in there – low-res images, vector layouts, floats, shapes, ads, etc).

In my spare time I hung around in forums to learn more about web development and print layouting/prepress. I got stuck with InDesign and started to try out new things on my own. Once, I attended a meetup in the big city 1hour away from where I lived (Munich). It was an InDesign user group and I was hooked. There were people teaching others cool things for free and you could discuss with people. After the event the organizers gave everyone a feedback form which contained also the question if one would be interested in speaking there, too. I don’t remember what lead me to mark the cross at "yes" but a few weeks later I got a call. Their original speaker turned the gig down and they needed a replacement. Clueless what I could talk about they asked me if I knew something about InDesign and XML. I said no but I could figure it out which was enough for them and that’s how I gave my first public talk and a few meetings later, joined the organization team for the user group (for like 5 years).

After my Abitur (university-entrance diploma) I worked a bit freelance and then started my civil service in Munich and moved there. During that, I received several requests for projects and did them in my spare time. It was the time HTML5 came around and I wanted to go back into web development. A few smaller websites followed and when I finished my 9 month of civil service I went full time freelance instead of going to university. I got more than enough projects to make a living from it and due to my blog articles, speaking appearances (still all local stuff) I apparently got enough attention that people shared that I do good work…

Basically, this is how I started. The story goes on but that’d be way too long now. Hope it helps. :) If you have questions on details, please ask.

**Comment by anselmh** [comment]
Actually, it seems you wanted to also know **how to become a freelancer**. A few tips from me:
- Start gaining interest by potential clients before going freelance (blog, speak, share something, open source work)
- Search yourself a corner that you’re good at and that you love to get work
- Research how much money you need for a living
- Calculate your hourly and daily rate based on that and charge a bit more than that
- Don’t assume to start out covering 100% of your costs in the first year, it’ll take some time to get enough good paid projects
- Have a portfolio and CV
- Open the legal business, care about your taxes in advance and get someone who can help you on that

And finally, get colleagues who can help you out on the development/design and get people who can help you out on the business side. It first feels weird to pay an accountant but be sure it’s worth the money.
Take this as basic advice, please. There might be some things missing now but I think the most important is to put yourself out on the market and be very disciplined on working on your own and managing finances.

**Comment by Daliprince** [comment]
Tips are really helpful, and  your story give me some direction, thank you for your detailed answer!
